channels/jp.m3u

```diff
@@ -18,11 +18,11 @@ https://livecdn.fptplay.net/hda3/animaxport_2000.stream/playlist.m3u8
 https://livecdn.fptplay.net/hda3/animaxport_hls.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="ATX.jp" tvg-name="AT-X" tvg-country="JP" tvg-language="Japanese" tvg-logo="https://www.at-x.com/favicon_android.png" group-title="Animation",AT-X
 https://sub2.neetball.net/live/neet.m3u8
+#EXTINF:-1 tvg-id="BSTVTokyo.jp" tvg-name="BS TV Tokyo" tvg-country="JP" tvg-language="Japanese" tvg-logo="" group-title="General",BS TV Tokyo
```

Comment:
This one is actually Japanet DX, streaming channels on daytime, and Japanet DX commercial announcements + shopping on nighttime.

Reply:
Interesting advertisments too.

https://www.shopjapan.co.jp/products/OTHR-80005/

![vlcsnap-2021-08-04-13h51m30s408](https://user-images.githubusercontent.com/30985701/128176158-44007328-cf15-4ff8-b2b6-535b01ee38e2.png)
![vlcsnap-2021-08-04-13h53m01s576](https://user-images.githubusercontent.com/30985701/128176165-77c1c105-88fa-48cd-bb26-2eb06a0c67b7.png)

Right now, Nikkei News 9 is being streamed.